tests_utils.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+
+import pytest
+
+from heuristics import (
+    get_column_type,
+    is_euro_format,
+    is_participant_id,
+    is_sex,
+    is_yes_no,
+)
+from utils import read_csv_autodetect_date
+
+
+@pytest.fixture
+def input_tsv():
+    return Path(__file__).parent / "tests" / "data" / "participants.tsv"
+
+
+def test_read_csv(input_tsv):
+    df = read_csv_autodetect_date(input_tsv, sep="\t")
+    assert df.acq_date.dtype == "datetime64[ns]"
+
+
+def test_is_yes_no(input_tsv):
+    df = read_csv_autodetect_date(input_tsv, sep="\t")
+
+    assert ~is_yes_no(df["acq_date"])
```

Comment:
This should probably be `not` instead of `~`. Otherwise I don't think this assertion can ever be false 

```python
>>> bool(~True)
True
```

Reply:
clearly a Matlabism caught in the wild in the middle of the pyton code...

that's happens when you keep switching between matlab and python... you mix up the syntaxes...

Reply:
ahh, yes, I vaguely remember now :)